pyvips/base.py

```diff
@@ -89,6 +89,24 @@ def type_map(gtype, fn):
     return vips_lib.vips_type_map(gtype, cb, ffi.NULL, ffi.NULL)
 
 
+def values_for_enum(gtype):
+    """Get all values for a enum (gtype)."""
+
+    g_type_class = gobject_lib.g_type_class_ref(gtype)
+    g_enum_class = ffi.cast('GEnumClass *', g_type_class)
+
+    values = []
+
+    # -1 since we always have a "last" member.
+    for i in range(0, g_enum_class.n_values - 1):
+        value = _to_string(ffi.string(g_enum_class.values[i].value_nick))
+        values.append(value)
+
+    glib_lib.g_free(g_enum_class)
```

Comment:
Are you sure you need this? `g_type_class_ref()` results can usually just be left: extra refs on classes don't matter, since we never unload., and I think freeing them is not correct.

Reply:
I wasn't sure about this one. Removed with https://github.com/jcupitt/pyvips/pull/47/commits/d27b8e229809cf1ca4985cd4609c56385a94a89c.

Thanks!